src/main/java/homework4/Bank.java

```diff
@@ -0,0 +1,71 @@
+package homework4;
+
+
+public class Bank<V extends Asset> {
+    private final String addr;
+    private Asset assets[];
+
+
+    public Bank(String addr) {
+        this.addr = addr;
+    }
+
+
+    public void withdraw(long amount, String title) {
+        for (int i = 0; i < assets.length; i++) {
+            if (assets[i].getClass().getSimpleName().equals(title)) {
+                assets[i].setQuantity(assets[i].getQuantity() - amount);
+                break;
+            }
+        }
+    }
+
+
+    public void put(V value) {
+        if (Bank.isAsset(value)) {
```

Comment:
On class level you wrote V extends Asset
Bank.isAsset (...instanseOf...) don't need